packages/design-system/src/components/web-components/shared-attributes/accessibility.ts

```diff
@@ -0,0 +1,50 @@
+// All the WAI-ARIA 1.1 attributes from https://www.w3.org/TR/wai-aria-1.1/
+
+export const ariaAttrs = [
+  'aria-activedescendant',
+  'aria-atomic',
+  'aria-autocomplete',
+  'aria-busy',
+  'aria-checked',
+  'aria-colcount',
+  'aria-colindex',
+  'aria-colspan',
+  'aria-controls',
+  'aria-current',
+  'aria-describedby',
+  'aria-details',
+  // 'aria-disabled', // This prop resulted in errors in Astro.
```

Comment:
Including `aria-disabled` resulted in rendering issues on Astro. It's not entirely clear to me why.

I tested this pass-thru logic locally on Astro's web components (ds-choice, ds-dropdown) and it placed the attrs where they ought to be. I used something crazy like `aria-checked="mixed"` so it would stand out.

Reply:
Oh - I also thought I would've needed to include this on ds-choice-list and ds-month-picker, but neither of those have the same prop spreading as the more atomic elements do.

Reply:
I'm actually not 100% sure we should be adding these aria ones to the attributes of the web component. What happens with assistive tech when both the web component root and one of its children has an `aria-label` on it that is the same? We might have to pass the aria attributes down using some other alias like how we have `root-id` that passes an `id`.

I think all the other lists are good, but this one might need more research.

Reply:
OK I'll keep the aria list and remove the imports from 🚾 so we can experiment in the future if we have time